src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -47,7 +48,11 @@ public async Task<int> ExecuteCommandAsync(VerifyArgs verifyArgs)
                         _defaultFingerprintAlgorithm)).ToList();
 
                 var verifierSettings = SignedPackageVerifierSettings.GetVerifyCommandDefaultPolicy();
-                var verificationProviders = SignatureVerificationProviderFactory.GetDefaultSignatureVerificationProviders();
+                var verificationProviders = new List<ISignatureVerificationProvider>()
+                {
+                    new IntegrityVerificationProvider(),
+                    new SignatureTrustAndValidityVerificationProvider()
```

Comment:
why dont we need to pass the `allowedUntrustedRoot` list here for verify command?

Reply:
Verify command does not have a way to get this information. We discussed in a standup if we wanted verify command to read from the config file the trusted signers (and therefore, also this information) and it was decided that it shouldn't for now, and if we have requests from customers we can easily add this in the future.

Reply:
Do/should we have an issue about that?

Reply:
In my opinion, if this is an ask someone has, they will open an issue eventually, and if not, there's no reason to track work we will never do.

Reply:
That's fine by me.